rollup/internal/orm/chunk.go

```diff
@@ -202,6 +204,7 @@ func (o *Chunk) InsertChunk(ctx context.Context, chunk *encoding.Chunk, codecVer
 		parentChunkStateRoot = parentChunk.StateRoot
 	}
 
+	fmt.Println("insertChunk", totalL1MessagePoppedBefore, chunkIndex, parentChunkHash)
```

Comment:
prefer using `log.Info`, it's better shown in grafana.

Reply:
Ah sure that's just a debug print actually :upside_down_face: